projects/1.20-fabric/assets/species/species/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+{
+  "itemGroup.species.species": "新奇物种",
+  "item.species.logo": "新奇物种",
+
+  "species.developer.made_by": "设计者:",
```

```suggestion
  "species.developer.made_by": "设计者：",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。